src/stores/dex.js

```diff
@@ -0,0 +1,57 @@
+import { defineStore } from 'pinia'
```

Comment:
Well... my interpretation of that conversation is that we didn't agree on anything specific apart from dealing with this kind of duplicates at some point. I just asked for your opinion about moving this to a composable since this code is stateless but before I got the response I ended up implementing it in this store in order not to implement any new concepts. In my opinion, this is not a new concept since stores like `aesdk`, `blockchainStats`, `marketStats` or `websocket` also don't belong to a specific page and are widely used in many places.

Reply:
I thought we concluded on leave it as it is, or moving it into format.js until we got more occurrences. So I am not sure what was the discussion for

Reply:
The discussion was supposed to be about moving it to a composable but I decided on my own how to deal with it before you have answered.

Reply:
Seems like the reused logic is now distributed on several places without a plan to improve it

Reply:
I would say in this specific case it's actually stored in one place rather than multiple but I guess it's subjective. Anyway I created a followup task to address this and other cases of duplicated code for API calls https://github.com/aeternity/aescan/issues/485